django_startr/admin/views.py

```
@@ -0,0 +1,110 @@
+        inline_class_name = getattr(inline, 'class_name', inline.__class__.__name__)
+        if inline_class_name.lower() == inline_type.lower():
+            inline_instance = inline
+            logger.debug(f"Found inline instance: {inline_instance}")
+            break
+    
+    if not inline_instance:
+        logger.error(f"Inline {inline_type} not found")
+        return JsonResponse({'error': f'Inline {inline_type} not found'}, status=404)
+    
+    # Create a context that mimics the admin change form context
+    admin_form = admin_instance.get_form(request, parent_obj)(instance=parent_obj)
```

Ruff (0.8.2)

Local variable `admin_form` is assigned to but never used

Remove assignment to unused variable `admin_form`

(F841)